scripts/check-release-contracts.mjs

```
@@ -106,17 +106,32 @@ const packDestinations = [...publishWorkflow.matchAll(/--pack-destination\s+([^\
+);
+assert.match(
+  publishWorkflow,
+  /path: \$\{\{ runner\.temp \}\}\/package-\$\{\{ github\.run_id \}\}-\$\{\{ github\.run_attempt \}\}\/\*\.tgz/,
+);
+assert.match(publishWorkflow, /find "\$PACKAGE_DIR" -maxdepth 1 -name "\*\.tgz"/);
 assert.match(publishWorkflow, /npm view .*version >"\$view_output" 2>&1/);
 assert.match(publishWorkflow, /grep -Eq .*E404\|404/);
 assert.match(publishWorkflow, /exit "\$view_status"/);
 assert.match(publishWorkflow, /\[\[ "\$package_version" == \*-\* \]\]/);
 assert.match(publishWorkflow, /dist_tag=next/);
 assert.match(publishJob, /^    environment: publish$/m);
```

ESLint

[error] Spaces are hard to count. Use {4}.

(no-regex-spaces)